### PR TITLE
Addressed issue #183 whereby a failure in the test always gets overwr…

### PR DESCRIPTION
**Pull request by niamccash**
Addressed issue #183 whereby a failure in the test always gets overwitten by success status regardless of actual status.

Developed on a New York version personal dev instance.

### CODE REVIEW

**Comment by niamccash** [comment]
@jacebenson  I'm a total noob here and could use some guidance.  Can I resolve these conflicts in my local repo via web UI or will it need to be done in the platform somehow? Seems I forgot to grab your latest version of the repository.

File: sys_app_29192d90db0123008096a455ca96193d.xml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?><record_update table="sys_app">
     <sys_app action="INSERT_OR_UPDATE">
         <active>true</active>
+        <can_edit_in_studio>false</can_edit_in_studio>
```
Comment (jacebenson) [inline sys_app_29192d90db0123008096a455ca96193d.xml]: Hmm.. Not sure about this, not sure if SN set this automatically, or you did.

Comment (niamccash) [reply sys_app_29192d90db0123008096a455ca96193d.xml]: The only thing I manually changed was the code in the Test Step.

**Comment by jacebenson** [comment]
Because SN depends on the entire repo being current.  You'll need to pull the repo into this branch and then do a commit/push because I cannot accept with any conflicts.

**Comment by jacebenson** [comment]
You should have an option to "Apply Remote Changes" in your branch.
image

Once you do that, touch something and re commit the changes.  Then I'm not sure if you need a new PR or if this PR will have the updated code.  But that's where I think this needs to be.

**Comment by niamccash** [comment]
Ok, so what I'm learning is that this might not work so well because the Table (sys_db_object) for Test Step (sys_atf_step), as highlighted in some of the conflicts, has a different sys_id in your instance than it does in mine. I have been working in my personal dev instance but I also checked my organizational instance and the sys_db_object there has yet a different sys_id. 

Thoughts?

**Pull request closed without merging**